domenetjenester/brevbestiller/src/main/java/no/nav/foreldrepenger/melding/datamapper/domene/BeregningsresultatMapper.java

```diff
@@ -114,14 +127,35 @@ private static PeriodeType mapEnkelUttaksperiode(UttakResultatPeriode uttakperio
         return periode;
     }
 
+    private static PeriodeType mapForstePeriode(BeregningsresultatPeriode beregningsresultatPeriode,
+                                                UttakResultatPeriode uttakResultatPeriode,
+                                                BeregningsgrunnlagPeriode beregningsgrunnlagPeriode) {
+        PeriodeType periodeMedFraDato = objectFactory.createPeriodeType();
+        if(uttakResultatPeriode.getFom().isBefore(beregningsresultatPeriode.getBeregningsresultatPeriodeFom())) {
+            periodeMedFraDato.setPeriodeFom(XmlUtil.finnDatoVerdiAvUtenTidSone(uttakResultatPeriode.getFom()));
+        }
+        else {
+            periodeMedFraDato.setPeriodeFom(XmlUtil.finnDatoVerdiAvUtenTidSone(beregningsresultatPeriode.getBeregningsresultatPeriodeFom()));
+        }
+        return mapEnkelPeriode(beregningsresultatPeriode, uttakResultatPeriode, beregningsgrunnlagPeriode, periodeMedFraDato);
+    }
+
+    private static PeriodeType mapPeriode(BeregningsresultatPeriode beregningsresultatPeriode,
+                                                UttakResultatPeriode uttakResultatPeriode,
+                                                BeregningsgrunnlagPeriode beregningsgrunnlagPeriode) {
+        PeriodeType periodeMedFraDato = objectFactory.createPeriodeType();
+        periodeMedFraDato.setPeriodeFom(XmlUtil.finnDatoVerdiAvUtenTidSone(beregningsresultatPeriode.getBeregningsresultatPeriodeFom()));
+        return mapEnkelPeriode(beregningsresultatPeriode, uttakResultatPeriode, beregningsgrunnlagPeriode, periodeMedFraDato);
+    }
+
     private static PeriodeType mapEnkelPeriode(BeregningsresultatPeriode beregningsresultatPeriode,
                                                UttakResultatPeriode uttakResultatPeriode,
-                                               BeregningsgrunnlagPeriode beregningsgrunnlagPeriode) {
-        PeriodeType periode = objectFactory.createPeriodeType();
+                                               BeregningsgrunnlagPeriode beregningsgrunnlagPeriode,
+                                               PeriodeType periode) {
```

Comment:
Hva med å sende inn en LocalDate fom i steden for PeriodeType? Da slipper vi å opprette PeriodeType i begge metodene.

Reply:
Har lagt inn LocalDate i stedet. Det første også til at jeg fjernet metoden mapPeriode - den hadde ingen funksjon da. Ser det ok ut?